### PR TITLE
Make plugins collapsible

### PR DESCRIPTION
**Pull request by dhaneshnm**
All plugins are collapsible. Admin plugin is collapsed by default.

### CODE REVIEW

**Comment by dhaneshnm** [comment]
Screenshot_2020-03-11 A knowledge graph for data-driven science(1)

**Comment by dhaneshnm** [comment]
Screenshot_2020-03-11 A knowledge graph for data-driven science

File: src/shared/containers/ResourcePlugins.tsx [hunk 1]
```diff
@@ -23,17 +25,27 @@ const ResourcePlugins: React.FunctionComponent<{
...
-            pluginName={pluginName}
-            resource={resource}
-            goToResource={goToResource}
-          />
-        </div>
+    <Collapse
+      defaultActiveKey={[...Array(filteredPlugins.length).keys()].map(i =>
+        (i + 1).toString()
+      )}
+      onChange={() => {}}
```
Comment (Stafeeva) [inline src/shared/containers/ResourcePlugins.tsx]: is it finished? :)

File: src/shared/containers/ResourceViewContainer.tsx [hunk 2]
```diff
@@ -241,6 +242,88 @@ const ResourceViewContainer: React.FunctionComponent<{
                   closable
                 />
               )}
+              <ResourceActionsContainer resource={resource} />
+              <Collapse defaultActiveKey={[]} onChange={() => {}}>
```
Comment (Stafeeva) [inline src/shared/containers/ResourceViewContainer.tsx]: same here - why do you pass () => {} as onChange?

File: src/shared/containers/ResourceViewContainer.tsx [hunk 3]
```diff
@@ -241,6 +242,88 @@ const ResourceViewContainer: React.FunctionComponent<{
...
+                <Panel header={'Admin plugins'} key="1">
+                  <Tabs activeKey={activeTabKey} onChange={handleTabChange}>
+                    <TabPane tab="JSON" key="#JSON">
+                      <ResourceEditorContainer
+                        resourceId={resource['@id']}
+                        orgLabel={orgLabel}
+                        projectLabel={projectLabel}
+                        rev={resource._rev}
+                        defaultExpanded={
+                          !!expandedFromQuery && expandedFromQuery === 'true'
```
Comment (Stafeeva) [inline src/shared/containers/ResourceViewContainer.tsx]: is true a string, not a boolean?

**Pull request merged**